### PR TITLE
Minor Changes

### PR DESCRIPTION
**Pull request by sarveshpro**
Changes in this pull request:
-  added open sans for cost button font

### CODE REVIEW

File: src/theme/_fonts.scss [hunk 1]
```diff
@@ -1,4 +1,8 @@
 @import url('https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
 
-$primaryFontFamily:'Raleway', sans-serif;
-$fontSubTitleSize:14px;
\ No newline at end of file
+$primaryFontFamily: 'Raleway', sans-serif;
+$fontSubTitleSize: 14px;
+
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap');
```
Comment (sagararyal) [inline src/theme/_fonts.scss:6]: we only need one variant maybe just weight 400.

File: src/theme/_fonts.scss [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+
+$openSans: 'Open Sans', sans-serif;
```
Comment (sagararyal) [inline src/theme/_fonts.scss:8]: font should have other system fallback fonts

Comment (sarveshpro) [reply src/theme/_fonts.scss:8]: importing only 400 done

**Comment by harshvitra** [comment]
Why aren't we merging this one ?

File: src/theme/_fonts.scss (hunk 1 above)
Comment (sarveshpro) [reply src/theme/_fonts.scss]: done

**Review by sagararyal — APPROVED** [review]

**Pull request merged**